integration/Jenkinsfiles/buildonly-daily.groovy

```diff
@@ -0,0 +1,83 @@
+
+properties([
+    [$class: 'BuildDiscarderProperty', strategy: [$class: 'LogRotator', daysToKeepStr: '60', numToKeepStr: '60', artifactNumToKeepStr: '1']],
+    disableConcurrentBuilds(),
+    [$class: 'GithubProjectProperty', displayName: '', projectUrlStr: 'https://github.com/nuxeo/'],
+    [$class: 'RebuildSettings', autoRebuild: false, rebuildDisabled: false],
+    parameters([booleanParam(defaultValue: true, description: '', name: 'CLEAN_REPO')]),
+    pipelineTriggers([cron('0 0 * * *')])
+])
+
+
+
+timestamps {
+    
+    timeout(time: 240, unit: 'MINUTES') {
+        
+        node ('SLAVE') {
+      
+            stage ('Checkout') {
+                checkout([$class: 'GitSCM', branches: [[name: 'master']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[credentialsId: '', url: 'https://github.com/nuxeo/nuxeo/']]])
+            }
+            stage ('Build') {
+            
+                withEnv(["MAVEN_OPTS=-Xmx2048m"]) {
+            
+                    // Shell Clean Workspace step
+                    sh """
+                        export PATH="/opt/build/tools/maven3/bin:$PATH"
+                        [ "$CLEAN_REPO" = "true" ] && rm -rf ${WORKSPACE}/.repository/org/nuxeo 2>/dev/null
+                        ./clone.py master 
+                       """        
+                    // Maven build step
+                    withMaven(jdk: 'java-11-openjdk', maven: 'maven-3', mavenOpts: '-Xms2g -Xmx4g', mavenLocalRepo: "$WORKSPACE/.repository") {
+                        sh "mvn -f pom.xml -V clean install -DskipTests=true -DskipITs=true -Paddons,distrib "
+                    }       
+                    // Maven build step
+                    withMaven(jdk: 'java-11-openjdk', maven: 'maven-3', mavenOpts: '-Xmx2g') {
+                        sh "mvn -f pom.xml -V versions:display-dependency-updates -nsu -N "
+                    }       
+                    // Maven build step
+                    withMaven(jdk: 'java-11-openjdk', maven: 'maven-3', mavenOpts: '-Xms2g -Xmx3g', mavenLocalRepo: "$WORKSPACE/.repository") {
+                        sh "mvn -f pom.xml -V license:aggregate-add-third-party -Paddons,distrib,release -nsu "
+                    }       
+                    // Maven build step
+                    withMaven(jdk: 'java-11-openjdk', maven: 'maven-3', mavenOpts: '-Xms2g -Xmx4g') {
+                        sh """
+                           mvn -V  \
+                           dependency:tree \
+                           -Paddons,distrib,release \
+                           -DoutputFile="${WORKSPACE}/target/generated-sources/dependency-tree.log" \
```

Comment:
`${WORKSPACE}/target` ? i'm not sure that the path is correct

Reply:
It is the same path as the maven job.
If it is not satisfying, what do you recommand ?

Reply:
You're right, it's consistent with https://qa.nuxeo.org/jenkins/job/master/job/nuxeo-master-buildonly-daily/ws/
We are used to put data in a folder instead of root. Nevermind, it's fine and consistent as is.
:+1: